docs/resource/reference.en-US.md

```diff
@@ -0,0 +1,65 @@
+---
+order: 1
+title: Reference
+---
+
+Below Books has inspired us, saved our time and helped us to overome difficulty when we design component and patterns. If you want to know more about design things, we recommend that you read these awesome books.
```

Comment:
Please find below the books that inspired us, saved our time and helped us to overcome difficulties when designing components and patterns. If you want to know more about UI design, we recommend you these awesome books.